### PR TITLE
NO-ISSUE Add customizations doc

### PR DESCRIPTION
**Pull request by carbonin**
This gives examples of all non-standard API-only customization options
for things like install config, ignitions, and manifests.

cc @ronniel1 @avishayt

### CODE REVIEW

**Comment by carbonin** [comment]
Not sure if we want more detail in this or just examples.

**Comment by avishayt** [comment]
I would add a sentence or two for each about what exactly it's customizing and when it can/should be used.

**Review by eranco74 — COMMENTED** [review]
/lgtm

File: docs/user-guide/install-customization.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+curl \
+    --header "Content-Type: application/json" \
+    --header "Authorization: Bearer $TOKEN" \
+    --request POST \
+    --data "{\"file_name\":\"$file\", \"content\":\"$content\"}" \
+"http://$ASSISTED_SERVICE_IP:$ASSISTED_SERVICE_PORT/api/assisted-install/v1/clusters/$CLUSTER_ID/manifests"
+```
+
+### View a manifest’s contents
```
[the remark this answers is not in the document]
Comment (carbonin) [reply docs/user-guide/install-customization.md:24]: Yeah, looks like I missed some endpoints https://github.com/openshift/assisted-service/blob/df6988c73f2f3a888fc199eec84d1fd3940d131e/swagger.yaml#L868-L869.

> consider adding info about how to get the authorization token

What token?

[the remark this answers is not in the document]
Comment (eranco74) [reply docs/user-guide/install-customization.md:24]: The token required for making the API call
https://github.com/openshift/assisted-service/pull/1268/files/71978e894541b2f884888a1b9cee2b48bdcabc84#diff-2ba87ee0005a7fb9cb20a9a3557045068d8e51e358692ed2cb52cb2feca321e0R18

[the remark this answers is not in the document]
Comment (carbonin) [reply docs/user-guide/install-customization.md:24]: Oh, right. I think that's probably worth its own doc as we have a few different auth mechanisms now.

**Pull request merged**